src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(Colors color,int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+
+        final double Pi = 3.1415926536;
+        return (Pi * radius * radius);
```

Comment:
```suggestion
        return (Math.PI * radius * radius);
```
можна використати існуючу константу